src/microservices/Microservices.CohortExtractor/Execution/RequestFulfillers/QueryToExecute.cs

```diff
@@ -125,33 +126,45 @@ public IEnumerable<QueryToExecuteResult> Execute(string valueToLookup, List<IRej
             using (DbConnection con = Server.GetConnection())
             {
                 con.Open();
-                DbDataReader r = Server.GetCommand(GetSqlForKeyValue(valueToLookup), con).ExecuteReader();
```

Comment:
Its fine this time but I think we should avoid renaming locals and making whitespace changes in commits if possible as it makes it harder to review exactly what is being done in the PR.

Reply:
Yeah I agree - PR changes should be minimal.

Here I wanted to clarify the `r` variable. In general, I think we would benefit from applying a linter like dotnet-format automatically to eliminate these sorts of changes (whitespace, code style etc.) needing to be discussed in PRs.